tests/util.rs

```diff
@@ -81,7 +81,7 @@ pub fn test_crl() -> (CertificateRevocationList, Certificate) {
 	let now = OffsetDateTime::now_utc();
 	let next_week = now + Duration::weeks(1);
 	let revoked_cert = RevokedCertParams{
-		serial_number: SerialNumber::from(9999),
+		serial_number: SerialNumber::from_slice(&[0xC0, 0xFF, 0xEE]),
```

Comment:
I switched to using `from_slice(&[u8])` here because `from(u64)` was converting the `u64` input into a big endian encoded slice that was padded out to the size of a `u64`, where the webpki side was returning a serial number `&[u8]` slice that had no padding. Changing to using a raw slice as input to the `RevokedCertParams` seemed like the simplest fix.